server/planning/autosave.py

```diff
@@ -20,6 +20,7 @@ class AutosaveService(Service):
     def on_create(self, docs):
         for doc in docs:
             AutosaveService._validate(doc)
+            doc.pop('expired', None)
```

Comment:
This is already handled in `AUTOSAVE.IGNORE_FIELDS`.

It would be best for the endpoint to simply save all the supplied data, leaving the client in control of what gets saved.

Reply:
'expired' is a default field is 'events' and 'planning' schema, so it comes up at the server side.